lib/msteams_hermes/message.rb

```diff
@@ -9,6 +9,29 @@ module MsTeamsHermes
   # https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/connectors-using?tabs=cURL#send-adaptive-cards-using-an-incoming-webhook
   ##
   class Message
+    ##
+    # Raises when the message is larger than the latest known maximum size of microsoft teams messages
```

Comment:
```suggestion
    # Raises when the message is larger than the latest known maximum size of a microsoft teams message
```
Just because we are in the context of sending one message via this instance and using singular creates a closer relation that current context.